src/Plugins/SimplnxCore/src/SimplnxCore/Filters/Algorithms/ReadVtkStructuredPoints.hpp

```diff
@@ -136,7 +136,7 @@ class SIMPLNXCORE_EXPORT ReadVtkStructuredPoints
    * @param length Length of line
    * @return Integer error value
    */
-  Result<> readLine(std::istream& in, char* result, size_t length);
+  static Result<> readLine(std::istream& in, char* result, size_t length);
```

Comment:
If these aren't used outside `ReadVtkStructuredPoints` then they could be moved into an anonymous namespace in the cpp file.